STMGeom/inc/SSCSupport.hh

```diff
@@ -0,0 +1,112 @@
+#ifndef STMGeom_SSCSupport_hh
+#define STMGeom_SSCSupport_hh
+
+// STM Collimator Object
+//
+// Author: Haichuan Cao
+// Sept 2023
+
+#include <string>
+
+#include "CLHEP/Vector/Rotation.h"
+#include "CLHEP/Vector/ThreeVector.h"
+
+namespace mu2e {
+
+  class SSCSupport {
+  public:
+
+    SSCSupport(bool build,
+                 double table_L, double table_H, double table_T,
+                 double leg_L, double leg_H, double leg_T,
+                 double base_L, double base_H, double base_T,
+                 double wall_L, double wall_H, double wall_T,
+                 double hole_H, double hole_T,
+                 double FLeadStand_L, double FLeadStand_H, double FLeadStand_T,
+                 double FLeadShim_H, double FLeadShim_T,
+                 double FAluminumShim_T, double FAluminumExtra_L, double FAluminumExtra_H,
+                 CLHEP::Hep3Vector const & originInMu2e = CLHEP::Hep3Vector(),
+                 CLHEP::HepRotation const & rotation = CLHEP::HepRotation()
+                 ) :
+      _build(build),
+      _table_L(table_L), _table_H(table_H), _table_T(table_T),
+      _leg_L(leg_L), _leg_H(leg_H), _leg_T(leg_T),
+      _base_L(base_L), _base_H(base_H), _base_T(base_T),
+      _wall_L(wall_L), _wall_H(wall_H), _wall_T(wall_T),
+      _hole_H(hole_H), _hole_T(hole_T),
+      _FLeadStand_L(FLeadStand_L), _FLeadStand_H(FLeadStand_H), _FLeadStand_T(FLeadStand_T),
+      _FLeadShim_H(FLeadShim_H), _FLeadShim_T(FLeadShim_T),
+      _FAluminumShim_T(FAluminumShim_T), _FAluminumExtra_L(FAluminumExtra_L), _FAluminumExtra_H(FAluminumExtra_H),
+      _originInMu2e(originInMu2e),
+      _rotation(rotation)
+    {
+    }
+
```

Comment:
This class only contains plain old data types (POD). Consider using a struct in future work, this will simplify the coding